app/services/volunteer_op_marker_builder.rb

```diff
@@ -0,0 +1,40 @@
+class VolunteerOpMarkerBuilder
+  def initialize(organisations, listener, marker_builder, url_helper)
+    @organisations = organisations
+    @marker_builder = marker_builder
+    @listener = listener
+    @url_helper = url_helper
+  end
+
+  def perform
+    marker_builder.build(organisations) do |org, marker|
+      marker.lat org.latitude
+      marker.lng org.longitude
+      marker.infowindow listener.render_to_string(partial: 'popup', locals: { org: org })
```

Comment:
Metrics/LineLength: Line is too long. [89/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)